### PR TITLE
feat(skin): update design tokens

### PR DESCRIPTION
**Pull request by tuentisre**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: src/skins/movistar-legacy.tsx [hunk 1]
```diff
@@ -46,10 +46,7 @@ export const palette = {
     darkModeGrey: '#242424',
 };
 
-/**
- * @deprecated Use `getMovistarSkin` instead.
- */
-export const getMovistarLegacySkin: GetKnownSkin = (variant) => {
```
Comment (atabel) [inline src/skins/movistar-legacy.tsx:52]: please revert this part

File: src/skins/movistar-legacy.tsx [hunk 2]
```diff
@@ -297,15 +296,5 @@ export const getMovistarLegacySkin: GetKnownSkin = (variant) => {
...
-
-    if (variant === 'prominent') {
-        skin.colors.brand = palette.movistarProminentBlue;
-        skin.colors.brandHigh = palette.movistarProminentBlueDark;
-        skin.colors.backgroundBrand = palette.movistarProminentBlue;
-        skin.colors.navigationBarBackground = palette.movistarProminentBlue;
-        skin.colors.backgroundFeedbackBottom = palette.movistarProminentBlue;
-        skin.colors.controlActivated = palette.movistarProminentBlue;
-        skin.colors.dividerInverse = palette.movistarProminentBlueDark;
-    }
```
Comment (atabel) [inline src/skins/movistar-legacy.tsx:309]: and this

File: src/skins/movistar-legacy.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/skins/movistar-legacy.tsx]: done

File: src/skins/movistar-legacy.tsx (hunk 2 above)
Comment (marcoskolodny) [reply src/skins/movistar-legacy.tsx]: done

**Pull request merged**